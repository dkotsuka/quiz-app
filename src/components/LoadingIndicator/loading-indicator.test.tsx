import React from 'react'
import { LoadingIndicator } from '.'
import {create} from 'react-test-renderer';

describe("LoadingIndicator", () => {
    it("matches snapshot", () => {
        const loading = create(<LoadingIndicator />)
        expect(loading).toMatchSnapshot()
    })
})