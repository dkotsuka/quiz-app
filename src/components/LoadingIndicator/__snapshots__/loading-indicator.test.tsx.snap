// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator matches snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "height": 400,
        "justifyContent": "center",
      },
    ]
  }
>
  <ActivityIndicator
    animating={true}
    color="#565656"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
